contrib/hbmxml/tests/tohash.prg

```diff
@@ -53,8 +53,13 @@ STATIC FUNCTION s_NodeToHash( pNode )
                   AAdd( hHash[ mxmlGetElement( pNode ) ], hHashChild )
                ENDIF
             ELSEIF hb_mxmlGetAttrsCount( pNode ) > 0
-               hHash[ mxmlGetElement( pNode ) + "@attr" ] := hb_mxmlGetAttrs( pNode )
+                  IF Empty( hHash[ mxmlGetElement( pNode ) ] ) 
+                        hHash[ mxmlGetElement( pNode ) ] := {}
```

Comment:
Vicktor, https://github.com/rafathefull/harbour-core/commit/a901a288eadb6cd5601bd83e1b19369ac9b3051e , because, i push not modified pull request closed. I upload it as zip
[tohash.prg.gz](https://github.com/vszakats/harbour-core/files/286122/tohash.prg.gz)